apps/chat/src/store/share/share.epics.ts

```diff
@@ -958,7 +958,7 @@ const deleteOrRenameSharedFolderEpic: AppEpic = (action$, state$) =>
       const isSharedFolder = folders.find(
         (folder) => folder.id === payload.folderId,
       )?.isShared;
-      const requireRevoke = payload.values ? payload.values.name : true;
+      const requireRevoke = 'values' in payload ? payload.values.name : true;
```

Comment:
Maybe revert this change? Previous was better because of typing i think

Reply:
it was done to fix this error:
![image](https://github.com/user-attachments/assets/a48b06ec-02d3-4396-a73d-23489fc3733b)